pages/docs/tutorials/generate-code.md

```diff
@@ -87,13 +76,13 @@ If you did not follow the previous tutorial and do not have an `asyncapi.yaml` f
 
 3. Go back to the previous terminal to check if your application logged the streetlight condition you just sent. You should see something like this displayed in the terminal:
     <CodeBlock language="bash">
-    {`light/measured was received:
+    {`lightMeasured was received from mosquitto:
```

Comment:
doesn't look like https://github.com/asyncapi/glee/issues/581 will be solved soon so you need to update instruction in step 2, to send test message to topic `lightMeasured` and not `light/measured`

Reply:
I don't think so. https://github.com/asyncapi/glee/pull/582 will most likely done this week. Leave it on me 👍

Reply:
awesome!

Reply:
done 👍